planning/planning_test_utils/include/planning_interface_test_manager/planning_interface_test_manager.hpp

```diff
@@ -222,8 +220,6 @@ class PlanningInterfaceTestManager
   void publishNominalPathWithLaneId(rclcpp::Node::SharedPtr target_node, std::string topic_name);
   void publishAbNominalPathWithLaneId(rclcpp::Node::SharedPtr target_node, std::string topic_name);
 
```

Comment:
@TakaHoribe 
this line should be deleted

Reply:
Thank you!